pkg/build/api/types.go

```diff
@@ -411,6 +411,8 @@ const (
 	// BuildConfigLabelDeprecated was used as BuildConfigLabel before adding namespaces.
 	// We keep it for backward compatibility.
 	BuildConfigLabelDeprecated = "buildconfig"
+	// BuildConfigDeletionAnnotation is an annotation that marks a BuildConfig that is being deleted
+	BuildConfigDeletionAnnotation = "openshift.io/build-config.deletion-pending"
```

Comment:
> @ironcladlou @deads2k does this mean that the dc controller should ignore altogether a deployment (that is explicitly marked as inert via a spec field) or the lack of a config change trigger is enough for the deployment to be considered inert?

I meant upstream.  I haven't heard that we'll be pulling it downstream, though given the upstream lack of image change triggers, I don' t see our DCs going away for some time.

I chose the example for vocabulary.